packages/core/src/scss/components/checkout/checkoutRemote/_checkoutRemote.scss

```diff
@@ -24,10 +24,11 @@
         }
     }
 
-    .gpay-button {
+    .gpay-button, .gpay-card-info-container {
         border-radius: $global-radius;
         height: $wallet-button-height;
         min-height: $wallet-button-height;
+        max-width: 240px;
```

Comment:
We have set a `$wallet-button-max-width:` as `320px` now, is this width also working for gpay?

Reply:
Yeah, just forgot to remove this param